Sources/PackageManagerDocs/Documentation.docc/IntroducingPackages.md

```diff
@@ -4,8 +4,48 @@ Learn to create and use a Swift package.
 
 ## Overview
 
-overview content here....
+A package consists of Swift source files, including the `Package.swift` manifest file. 
+The manifest file, or package manifest, defines the package's name and its contents using the PackageDescription module.
 
-### First Section
+Each package declares `Products`, a list of what the package produces.
+Types of products include libraries, executables, and plugins:
 
-First section content
+- A library defines a module that can be imported by other Swift code. 
+- An executable is a program that can be run by the operating system.
+- A plugin is executable code that the Swift Package Manager may use to provide additional commands or build capabilities.
+
+The package can also declare `Dependencies`, most frequently other swift packages that provide modules you use.
+Dependencies can also provide macros, plugins, or reference system or binary (non-source) dependencies.
+
+Each product is made up of one or more `Targets`, the basic building block of a Swift package.
+Each target specifies an output, may declare one or more dependencies on other targets within the same package and on products vended by the package’s dependencies.
+A target may define a library, a test suite, an executable, an macro, a binary dependency, and so on.
+
+### About Modules
+
+Swift organizes code into _modules_. 
+Each module specifies a namespace and enforces access controls on which parts of that code can be used outside of that module.
+
+A program may have all of its code in a single module, or it may import other modules as _dependencies_. 
+Aside from the handful of system-provided modules, such as Darwin on OS X or GLibc on Linux, most dependencies require code to be downloaded and built in order to be used.
```

Comment:
We must say "macOS" nowadays, not "OS X"

Reply:
Also, Glibc not GLibc